server/notification-providers/pagertree.js

```diff
@@ -0,0 +1,91 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { UP, DOWN, getMonitorRelativeURL } = require("../../src/util");
+const { setting } = require("../util-server");
+let successMessage = "Sent Successfully.";
+
+class PagerTree extends NotificationProvider {
+    name = "PagerTree";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        try {
+            if (heartbeatJSON == null) {
+                const title = "[Test] Uptime Kuma Alert";
+                return this.postNotification(notification, title, monitorJSON, heartbeatJSON);
```

Comment:
If `heartbeatJSON == null`, it is not for test. It is for general messages. You should send `msg`.

Reply:
Thank you @louislam!

I am not sure where the interface documentation is, so I was just looking at the PagerDuty integration for reference.

I have made your requested changes.